addons/binding/org.openhab.binding.allplay/src/main/java/org/openhab/binding/allplay/handler/AllPlayHandler.java

```diff
@@ -410,9 +410,9 @@ private void updateCurrentItemState(PlaylistItem currentItem) {
         try {
             updateState(CURRENT_USER_DATA, new StringType(String.valueOf(currentItem.getUserData())));
         } catch (SpeakerException e) {
-            logger.warn("Unable to update current user data: {}", e.getMessage(), e);
+            logger.warn("Unable to update current user data: ", e);
```

Comment:
When you add a detector as the [ManualMessageDetector](https://github.com/KengoTODA/findbugs-slf4j/blob/master/bug-pattern/src/main/meta/findbugs.xml#L13) it can report more then one bug patters - in this case the SLF4J_MANUALLY_PROVIDED_MESSAGE,SLF4J_FORMAT_SHOULD_BE_CONST.

They can be later filtered.